Mathlib/Algebra/Polynomial/CoeffMem.lean

```diff
@@ -101,4 +104,20 @@ lemma coeff_divByMonic_mem_pow_natDegree_mul (p q : S[X])
     gcongr <;> exact sup_le (by simpa) (by simpa [Submodule.span_le, Set.range_subset_iff])
   · simp
 
+open Ideal in
+lemma idealSpan_range_update_divByMonic [DecidableEq ι] (v : ι → R[X]) {i j : ι} (hij : i ≠ j)
+    (hi : (v i).Monic) :
+    span (Set.range (Function.update v j (v j %ₘ v i))) = span (Set.range v) := by
```

Comment:
Maybe we could have a version with `v j %ₘ v i` replaced by `v j + p • v i`, and it could fit around [Submodule.span_insert_eq_span](https://leanprover-community.github.io/mathlib4_docs/Mathlib/LinearAlgebra/Span/Defs.html#Submodule.span_insert_eq_span).

Reply:
I don't really see how to prove this more efficiently using `Submodule.span_insert_eq_span`, but I did extract your `v j + p • v i` version as a lemma

Reply:
Yeah I also don't see how, but I think they're similar in spirit, and unfortunately I didn't find more helpful lemmas around it.